recipes/py-lz4framed/meta.yaml

```diff
@@ -0,0 +1,48 @@
+# py-lz4framed (Iotic Labs)
```

Comment:
@ocefpaf: Sorry, one more question: To make a change like this, should I:
- Submit a pull request (via branch within feedstock without fork?) **or**
- Commit directly to the feed stock

Since this is a comment removal change only, should I still increment the build number or leave everything else as is?